src/app.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask
+
+def create_app(config_file):
+    app = Flask(__name__)
+    app.config.from_pyfile(config_file)
+
+    from models import db, migrate
```

Comment:
Reason you're doing the imports as you go rather than putting them all at the top? Just wanna figure out styling consistency.